entity/scenario_intersection/include/scenario_intersection/intersection.h

```diff
@@ -94,6 +96,7 @@ class Intersection
                     target_, boost::lexical_cast<std::string>(each), false);
               });
         }
+        return false;
```

Comment:
Here a return value was missing. I added `false` but am not sure if it's correct. @yamacir-kit as the author of this package, could you let me know what this value should be?

Reply:
@nnmm Your change is correct :+1:

Reply:
Thank you!